setup.py

```diff
@@ -21,9 +21,7 @@
     author="Greg Caporaso and Benjamin Callahan",
     author_email="gregcaporaso@gmail.com",
     description="Apply DADA2 to generate denoised sequence variants. ",
-    scripts=['q2_dada2/assets/run_dada_single.R',
-             'q2_dada2/assets/run_dada_paired.R',
-             'q2_dada2/assets/run_dada_ccs.R'],
+    scripts=['q2_dada2/assets/run_dada_single_paired_ccs.R'],
```

Comment:
I'd recommend renaming this R file as `run_dada.R` - `run_dada_single_paired_ccs.R` is a bit confusing to read initially.